extensions/amp-analytics/0.1/visibility-model.js

```diff
@@ -71,6 +78,11 @@ export class VisibilityModel {
     });
 
     this.eventPromise_.then(() => {
+      if (!this.onTriggerObservable_) {
+        dev().warn(TAG_, 'onTriggerObservable_ is unexpectedly null. URL=' +
+            this.ampdoc_.win.location.href);
```

Comment:
@bradfrizzell Any inabox-specific risks here?

Reply:
Nothing that stands out to me, but hongfei would probably be a better perspective

Reply:
Ok, thanks. Was just curious because this.ampWin_ in https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/iframe-transport.js is set by "this.isInabox_ ? this.win : this.getAmpDoc().win" in https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/amp-analytics.js#L348